src/pykeen/training/callbacks.py

```diff
@@ -490,6 +491,8 @@ class ValidationLossTrainingCallback(TrainingCallback):
     def __init__(
         self,
         triples_factory: CoreTriplesFactory,
+        callbacks: TrainingCallbackHint = None,
+        callback_kwargs: TrainingCallbackKwargsHint = None,
```

Comment:
might as well fix the name of this while we're here

Reply:
or not, if it's gonna be a big diff

Reply:
583cf16